pkg/reconciler/openshift/rbac/rbac.go

```diff
@@ -274,3 +315,41 @@ func createSA(ctx context.Context, saInterface v1.ServiceAccountInterface, ns st
 
 	return sa, nil
 }
+
+func createTrustedCABundleConfigMap(ctx context.Context, cfgInterface v1.ConfigMapInterface, name, ns string) error {
+	c := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+			Labels: map[string]string{
+				// user-provided and system CA certificates
+				"config.openshift.io/inject-trusted-cabundle": "true",
+			},
+		},
+	}
+
+	_, err := cfgInterface.Create(ctx, c, metav1.CreateOptions{})
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+	return nil
+}
+
+func createServiceCABundleConfigMap(ctx context.Context, cfgInterface v1.ConfigMapInterface, name, ns string) error {
+	c := &corev1.ConfigMap{
```

Comment:
Why are we not reading the configmap yaml?

Reply:
as above...